exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/AbstractMapVector.java

```diff
@@ -173,7 +173,7 @@ public ValueVector getChildByOrdinal(int id) {
    *
    * Note that this method does not enforce any vector type check nor throws a schema change exception.
    */
-  protected void putChild(String name, ValueVector vector) {
+  public void putChild(String name, ValueVector vector) {
     putVector(name, vector);
```

Comment:
If we're making this more public, isn't there some way we can verify that what is being loaded is right? Also, why isn't protected enough?